corehq/apps/accounting/static/accounting/js/credits.js

```diff
@@ -37,24 +37,38 @@ hqDefine('accounting/js/credits', [
         self.products = products;
         self.features = features;
         self.paymentHandler = paymentHandler;
-        self.general_credit = ko.observable(generalCreditItem(paymentHandler));
+        self.cc_general_credit = ko.observable(ccGeneralCreditItem(paymentHandler));
+        self.invoice_general_credit = ko.observable(invoiceGeneralCreditItem(paymentHandler));
 
         self.triggerPayment = function (paymentMethod) {
             self.paymentHandler.reset();
             self.paymentHandler.paymentMethod(paymentMethod);
             self.paymentHandler.costItem(prepaymentItems({
                 products: self.products,
                 features: self.features,
-                general_credit: self.general_credit,
+                general_credit: self.cc_general_credit,
+                invoice_credit: self.invoice_general_credit,
             }));
         };
         return self;
     };
 
-    var generalCreditItem = function (paymentHandler) {
+    var ccGeneralCreditItem = function (paymentHandler) {
```

Comment:
what does the `cc` stand for? I'm not super thrilled with the abbreviation here because it's not super clear what it means...

Reply:
That's fair, it's meant to be "credit card". I'll look at better naming along with snake_case/camelCase fixes.

Reply:
https://github.com/dimagi/commcare-hq/pull/36064/commits/4a6076ed8793992dfa1ffca5833eba97eeb02ebc